crates/consensus/common/src/validation.rs

```diff
@@ -336,7 +337,7 @@ pub fn validate_header_regarding_parent(
             } else {
                 // This BaseFeeMissing will not happen as previous blocks are checked to have them.
                 parent
-                    .next_block_base_fee(chain_spec.base_fee_params)
+                    .next_block_base_fee(chain_spec.base_fee_params(parent.timestamp + 12))
```

Comment:
12 seconds is an arbitrary number in the context of EL (or rather a best guess). in case of missed slots, the blocks can be further apart. since we know the child timestamp here, we should use it instead
```suggestion
                    .next_block_base_fee(chain_spec.base_fee_params(child.timestamp))
```